content/docs/1.7.0/advanced-resources/system-backup-restore/backup-longhorn-system.md

```diff
@@ -42,7 +42,7 @@ It includes below resources associating with the Longhorn system:
 
 > **Note:** Longhorn does not backup `Nodes`. The Longhorn manager on the target cluster is responsible for creating its own Longhorn `Node` custom resources.
 
-> **Note:**  Longhorn system backup bundle only includes resources operated by Longhorn.  
+> **Note:**  Longhorn system backup bundle only includes resources operated by Longhorn.
```

Comment:
@mantissahz What does "operated" mean here? Used? Controlled? Managed? I'm asking because the use of "operated" is strange.

Reply:
I would say `Managed` is the closer one. cc @c3y1huang

But it's my bad, my editor automatically changes the document and I only remove one line.